src/test/java/com/znsio/api/posts_albums_response.json

```diff
@@ -0,0 +1,121 @@
+[
```

Comment:
remove this file. PR should be clean and shouldn't contain any unnecessary file.

Reply:
ok sure